nominees/gcompris.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "GCompris",
+  "description": "GCompris is a high quality educational software suite, including a large number of activities for children aged 2 to 10. The list of activities include computer discovery, reading, arithmetic, Braille, science, etc.",
+  "website": "https://gcompris.net/index-en.html",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/gcompris/GCompris-gtk/blob/master/COPYING"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceURL": "http://www.educational-freeware.com/freeware/gcompris.aspx"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceURL": "https://www.youtube.com/watch?v=a15ayDOwlls"
```

Comment:
@rachita19082 could you kindly also add this [link](https://www.gnu.org/education/edu-software-gcompris.en.html) to supplement the video in Spanish. Thanks.

Reply:
@nathanbaleeta I have added the link in the same field separated by a comma in commit 9144424. Please let me know if that is correct.